tests/main.test.js

```diff
@@ -73,6 +73,13 @@ test('cares only about external CSS files', async () => {
   expect(finalCss).toEqual('.external{color:red}');
 });
 
+test('cares about style tags and external CSS files', async () => {
+  const { finalCss } = await runMinimalcss('css-in-js', {
+    styletags: true
+  });
+  expect(finalCss).toEqual('.cssinjs1,.external,.inline{color:red}');
```

Comment:
idea is to take into account inline styles so we would know if we can safely remove `@font` and `@keyframes`, but not output as part of css, because those styles already part of html.

Reply:
Can you add a comment here to explain the awesomeness. 
It could be something like:
```
// The css-in-js.html fixture has an external stylesheet and a <style> 
// tag. The <style> tag contains the selector `.inline{...}`.
// This test asserts that the selectors from the inline <style> tags really
// are included in the final CSS. 

```